### PR TITLE
PBI 128: Create Account Schema and Migration

### PR DESCRIPTION
**Pull request by edutjie**
## Describe your changes
- Create Account Schema
- Create Account table migration

## Issue ticket number and link
- ADU-128: https://linear.app/adudu/issue/ADU-128/account-db-schema

### CODE REVIEW

File: migrations/20241009070631_create_account_table.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE account (
+    id SERIAL PRIMARY KEY,
```
Comment (valeelim) [inline migrations/20241009070631_create_account_table.sql:4]: make this a varchar just like the rest

Comment (edutjie) [reply migrations/20241009070631_create_account_table.sql:4]: Oke noted

**Review by valeelim — APPROVED** [review]
LGTM

File: internal/account/account.go [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package account
+
+import "time"
+
+type Account struct {
+	ID        int       `json:"id"`
+	Email     string    `json:"email"`
+	Password  string    `json:"-"`
+	CreatedAt time.Time `json:"created_at"`
```
Comment (ferryhieronimus) [inline internal/account/account.go:9]: Should we add UpdatedAt too?

File: internal/account/account.go [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package account
+
+import "time"
+
+type Account struct {
+	ID        int       `json:"id"`
```
Comment (ferryhieronimus) [inline internal/account/account.go:6]: Why use int for ID?

File: internal/account/account.go (hunk 2 above)
Comment (edutjie) [reply internal/account/account.go:9]: Sure, I'll open another PR to add it

File: internal/account/account.go (hunk 3 above)
Comment (edutjie) [reply internal/account/account.go:6]: sorry I missed this one, willl correct it in the next PR

**Pull request merged**